StreamChatClient_v3/Database/DTOs/MemberModelDTO.swift

```diff
@@ -82,7 +82,9 @@ extension NSManagedObjectContext {
 
 extension MemberModel {
     static func create(fromDTO dto: MemberDTO) -> MemberModel {
-        let extraData = dto.user.extraData.flatMap { try? JSONDecoder.default.decode(ExtraData.self, from: $0) }
+        // It's safe to use `try!` here, because the extra data payload comes from the DB, so we know it must
+        // be a valid JSON payload, otherwise it wouldn't be possible to save it there.
+        let extraData = try! JSONDecoder.default.decode(ExtraData.self, from: dto.user.extraData)
```

Comment:
Why force here?

Reply:
I tried to explain it in the comment above its usage. Here, we just decode the data we've already saved to the DB, so we're sure they're valid. We check for `ExtraData` validity in the parsing step already.
If you don't like the `try!` and I put more code in with some "this should never happen" case.

Reply:
@b-onc ?

Reply:
I've checked the PR too quickly so I didn't see the comment! 🤦 
If you're certain there're no edge cases/race conditions that might cause this to crash, it's good!